context/src/main/java/io/micronaut/scheduling/ScheduledExecutorTaskScheduler.java

```diff
@@ -54,27 +56,35 @@ public ScheduledExecutorTaskScheduler(@Named(TaskExecutors.SCHEDULED) ExecutorSe
     }
 
     @Override
-    public ScheduledFuture<?> schedule(String cron, Runnable command) {
-        if (StringUtils.isEmpty(cron)) {
-            throw new IllegalArgumentException("Blank cron expression not allowed");
-        }
-        check("command", command).notNull();
-
-        NextFireTime delaySupplier = new NextFireTime(CronExpression.create(cron));
-        return new ReschedulingTask<>(() -> {
+    public ScheduledFuture<?> schedule(String cron, String timezone, Runnable command) {
+        return schedule(cron, timezone, () -> {
             command.run();
             return null;
-        }, this, delaySupplier);
+        });
     }
 
     @Override
-    public <V> ScheduledFuture<V> schedule(String cron, Callable<V> command) {
+    public <V> ScheduledFuture<V> schedule(String cron, String timezone, Callable<V> command) {
```

Comment:
This is a public method and thus cannot be changed. A new method will have to be created instead